authors/apps/core/models.py

```diff
@@ -0,0 +1,12 @@
+# from django.db import models
+
+
+# class TimestampedModel(models.Model):
+#     # A timestamp representing when this object was created.
+#     created_at = models.DateTimeField(auto_now_add=True)
+
+#     # A timestamp reprensenting when this object was last updated.
+#     updated_at = models.DateTimeField(auto_now=True)
+
+#     class Meta:
+#         abstract = True
```

Comment:
If this entire module is no longer needed, I think it's better to get rid of it than having commented code.
Not a good practice to push commented code on the repository.